deploy/installer/generated/ibm-block-csi-operator.yaml

```diff
@@ -1,18 +1,16 @@
----
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.4.1
-  creationTimestamp: null
```

Comment:
so why did we have it if we don't need it?

Reply:
I don't know